### DISCUSSION TITLE
use certificates from windows certificate store

### DISCUSSION THREAD

**Discussion by andreig1** [edenhill/librdkafka#3819] [Q&A] [answered]
Currently we use certificates from files, example:
  ssl.ca.location=C:\temp\kafka-certs\letsencrypt-pub.crt
  ssl.keystore.location=C:\temp\kafka-certs\bhj.p12
  ssl.keystore.password=XXXXX

How to configure kafka to use certificates from windows store?

**Comment by edenhill** [comment, marked as answer]
The Windows Cert store is currently only used for retrieving the CA chain (e.g., `ssl.ca.location` counterpart), not for client certificates.

**Reply by andreig1** [reply]
Should I read your answer as NO? Are you planning to add that functionality?

**Reply by edenhill** [reply]
Correct, not currently supported, but we might add it in the future if there's enough interest.

In the meantime you can read the certs from the cert store yourself and then pass them in memory to librdkafka through the `rd_kafka_conf_set_ssl_cert()` function.

**Reply by andreig1** [reply]
Count us as interested

**Reply by zlsmith86** [reply]
I am interested as well

**Reply by rz-ftnt** [reply]
@edenhill Is there an update on this? Almost 3 years :-)
